mol-loader/rdkit_utils.py

```diff
@@ -0,0 +1,79 @@
+from rdkit import Chem
+from rdkit.Chem import rdMolDescriptors
+from rdkit.Chem import Descriptors
+from rdkit.Chem.MolKey import InchiInfo
+
+INTERESTING_DESCRIPTORS = dict(
+    ExactMolWt=Descriptors.ExactMolWt,
+    FractionCSP3=Descriptors.FractionCSP3,
+    HeavyAtomCount=Descriptors.HeavyAtomCount,
+    LabuteASA=Descriptors.LabuteASA,
+    MolLogP=Descriptors.MolLogP,
+    MolWt=Descriptors.MolWt,
+    NHOHCount=Descriptors.NHOHCount,
+    NOCount=Descriptors.NOCount,
+    NumAliphaticCarbocycles=Descriptors.NumAliphaticCarbocycles,
+    NumAliphaticHeterocycles=Descriptors.NumAliphaticHeterocycles,
+    NumAliphaticRings=Descriptors.NumAliphaticRings,
+    NumAromaticCarbocycles=Descriptors.NumAromaticCarbocycles,
+    NumAromaticHeterocycles=Descriptors.NumAromaticHeterocycles,
+    NumAromaticRings=Descriptors.NumAromaticRings,
+    NumHAcceptors=Descriptors.NumHAcceptors,
+    NumHDonors=Descriptors.NumHDonors,
+    NumRotatableBonds=Descriptors.NumRotatableBonds,
+    NumSaturatedCarbocycles=Descriptors.NumSaturatedCarbocycles,
+    NumSaturatedHeterocycles=Descriptors.NumSaturatedHeterocycles,
+    NumSaturatedRings=Descriptors.NumSaturatedRings,
+    RingCount=Descriptors.RingCount,
+    TPSA=Descriptors.TPSA,
+    NumAmideBonds=rdMolDescriptors.CalcNumAmideBonds,
+    NumBridgeheadAtoms=rdMolDescriptors.CalcNumBridgeheadAtoms,
+    NumSpiroAtom=rdMolDescriptors.CalcNumSpiroAtoms
+)
+
+
+def rdkit_descriptors(mol):
+    """
+    Given a molecule, return a dict of lots of interesting descriptors.
+
+    :param mol: the molecule to process
+    """
+    dd = {}
+    for name, fn in INTERESTING_DESCRIPTORS.items():
+        dd[name] = fn(mol)

```

Comment:
These three lines could be:
`dd = {k:fn(m) for k,fn in INTERESTING_DESCRIPTORS.items()}`